Rakefile

```diff
@@ -15,6 +17,48 @@ task :default => [:spec]
 
 Rake::Task["db:structure:dump"].clear unless Rails.env.development?
 
+def connect_sequel
```

Comment:
should probably be `def self.connect_sequel`, otherwise this method is defined on `Object`.